rj_gameplay/rj_gameplay/tactic/seek.py

```diff
@@ -13,20 +14,21 @@ def __init__(
         self,
         world_state: stp.rc.WorldState,
         num_seekers: int,
-        formations: List,
-        centroids: List,
+        formations: stp.formations.Formations,
     ):
         super().__init__(world_state)
 
-        formation = formations
-        centroid_list = centroids
+        formation = formations.get_regions
+        centroid_list = formations.get_centroids
         self._used_regions = []
         self._used_centroids = []
         self._num_seekers = num_seekers
-
+        # TODO: make seeker be able to handle more robots than formation regions available OR make formations have more regions
```

Comment:
```suggestion
        # TODO (Prahbanjan): make seeker be able to handle more robots than formation regions available OR make formations have more regions
```
Can you make this TODO a part of this PR? If we're replacing all unassigned behavior with seekers, having some hardcoded limit of seekers based on the formation will make developing new plays more confusing.

Reply:
The current way that it's being handled is the only way to "handle more robots than there are regions". The end result is that multiple robots are being passed into the same region. The only viable solution is to either:

A) add more regions to the formations class that are smaller and more definite 
B) create some logic in the seeker role to avoid being a certain distance from our robots in that region

The first one is less related to this PR. The second one makes the seeker role clunky. Will need to discuss this issue further, which is why the TODO was added instead of fixing the issue.

Reply:
okay in that case, please add a comment after the TODO that explains why the clunky-looking reset code